src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsactivitiesmanagementapi/service/AppointmentService.kt

```
@@ -66,6 +66,8 @@ class AppointmentService(
     val startTimeInMs = System.currentTimeMillis()
     val now = LocalDateTime.now()
 
+    request.failIfCategoryIsVideoLink()
```

Sorry Nagendra, this is not quite right.

This reads as fail if the category code is a video link.  It should be more a long the lines of fail if category code is video link and extra info is missing e.g. 

`request.failIfCategoryIsVideoLinkAndMissingExtraInfo()`